packages/app/src/systems/AppConnect/methods.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtensionPageConnection } from '@fuels-wallet/sdk';
+
+import type { AppConnectMachineService } from './machines';
+
+import { waitForState } from '~/systems/Core/utils/machine';
+
+export class AppConnectExternal extends ExtensionPageConnection {
```

Comment:
the idea is to be a common name like `routes.ts` it's possible this will be change when we implement more things like `signMessage` and `sendTransaction`. Let's move like this and change after?